besspin/cyberPhys/commander.py

```diff
@@ -176,13 +182,14 @@ def ready_enter(self):
         TODO: when to reset ALL targets?
         """
         # Periodically send CMD_COMPONENT_READY()
-        if (time.time() - self.last_ready_msg) > self.READY_MSG_TIMEOUT:
-            self.sendComponentReady(canlib.BESSPIN_TOOL)
```

Comment:
What replaces this timeout mechanism?

Reply:
Currently nothing is consuming `CMD_COMPONENT_READY` message so ti was cluttering the debug prints. We would like to use that info *eventually* though

Reply:
Makes sense.